uint16 msecs_to_reset
---
uint8 confirm